packages/client/src/schedule-types.ts

```diff
@@ -19,7 +19,7 @@ export interface ScheduleOptions<A extends ScheduleOptionsAction = ScheduleOptio
   /**
    * When Actions should be taken
    */
-  spec: RequireAtLeastOne<ScheduleSpec, 'calendars' | 'intervals' | 'cronExpressions'>;
+  spec: ScheduleSpec;
```

Comment:
I'd keep the old definition and make it optional.

Reply:
This has to match `ScheduleDescription` to maintain the `update` callback signature. See above comment on why I think it is better that spec be non-optional.